published/040-2019-apr-30.md

```diff
@@ -0,0 +1,66 @@
+
+Looking to get involved? Click on some of the links below, see what we’re up to on [GitHub](https://github.com/ipfs), or join us on [IRC](https://riot.im/app/#/room/#ipfs:matrix.org).
+
+Want this update in your inbox? [Subscribe to our weekly newsletter!](https://tinyletter.com/ipfsnewsletter)
+
+Here are some of the highlights since the last IPFS Weekly.
+
+
+## The latest
+
+###
+
+###
+
+## Meet [CONTRIBUTOR]
+
+Meet a person!
+
+
+
+## IPFS in the wild
+*Do you follow [IPFS on Twitter](https://twitter.com/IPFSbot)? For the latest mentions of IPFS in the news, check our Twitter feed or see the [latest articles on Awesome IPFS](https://awesome.ipfs.io/categories/articles/).* 
+
++ [Unstoppable Domains and the End of Internet Censorship](https://medium.com/hedgetradehq/unstoppable-domains-and-the-end-of-internet-censorship-241feff0b253), HedgeTrade, 22 Apr 2019
++ VIDEO: [Vaibhav Saini of Towards Blockchain - InterPlanetary File System (IPFS)](https://www.youtube.com/watch?time_continue=3&v=MvGJhlvrQlA), Let’s Talk ETC! 19 Apr 2019
++ VIDEO: [Where is the data bazaar?](https://www.youtube.com/watch?time_continue=1745&v=GcPU1Os2-Q8) Kris Skrinak, 10 Apr 2019
++ [Space-Ready Publishing with Gatsby and IPFS](https://interplanetarygatsby.com/foreword/), Interplanetary Gatsby, 31 March 2019
+
+
+## Updates and new releases
+*See the latest releases of IPFS tools and projects across the ecosystem.*
+
++ 
+
+## Questions from the community 🤔
+*Here are some questions folks are discussing in the IPFS ecosystem.*
+
++ [
+
+## Tools and projects we <3
+*[Awesome IPFS](https://awesome.ipfs.io/) is a community maintained and updated list of projects, tools, or pretty much any things related to IPFS that are totally awesome. To see more, or add yours to the list, visit [Awesome IPFS on GitHub](https://github.com/ipfs/awesome-ipfs).* 
+
++ [
+
+![](https://ipfs.io/ipfs/Qmd11gtyigpCjo4MfzXuj9MKuMF3Dj1EZEvbNRZeQE1jd4)
+
+## Join us at the first ever IPFS Camp
+
+[IPFS Camp](https://blog.ipfs.io/72-ann-ipfs-camp/) is taking place June 27th-30th in beautiful Barcelona. Meet the core developers and contributors to the IPFS project, community leaders, and builders on of the Distributed Web, but only if you register to attend. [Registration is open now!](https://camp.ipfs.io/)
+ 
+ 
+## Coming up in the Community
+*Did you know IPFS has a community forum at [discuss.ipfs.io](https://discuss.ipfs.io/)? Sign up to participate in discussions about coding, tutorials, see announcements and learn about upcoming community events.*
+
+
++ **2 May 2019:** [Munich IPFS User Group](https://www.meetup.com/de-DE/Munich-IPFS-User-Group/events/259762490/): [Pierre Krieger](https://twitter.com/tomaka17) from [parity](https://www.parity.io/) talks about Rust libp2p.
++ **2-3 May 2019:** [Uphill Conf](https://uphillconf.com/) – The inspiring frontend conference on top of beautiful mount Gurten, Bern Switzerland.
```

Comment:
Could we get an announcement for an SF IPFS Meetup on May 9th?

Details:

* Where: 2169 Folsom Street, Apt M202
* When: May 9th, 6pm-9pm
* What: Come hack on IPFS.

Reply:
@Stebalien Do you have a link we can share here?